eval/Containerfile

```diff
@@ -1,18 +1,24 @@
 FROM nvcr.io/nvidia/cuda:12.4.1-devel-ubi9
 
-RUN dnf install -y python3.11 python3.11-devel git python3-pip make automake gcc gcc-c++ \
+ARG EVAL_VERSION=v0.2.1
+ARG VLLM_VERSION=2024.08.01
+
+RUN dnf install -y python3.11 python3.11-devel python3-pip \
+    git make automake gcc gcc-c++ \
+    libcudnn8 libcudnn8-devel \
     && dnf clean all \
     && rm -rf /var/cache/*dnf*
 
-# TODO update to instructlab/eval after https://github.com/instructlab/eval/pull/128 or equivalent merges
+ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/local/lib/python3.11/site-packages/nvidia/cudnn/lib:/usr/local/cuda-12.4/compat"
+
+ENV TORCH_CUDA_ARCH_LIST="5.0;8.0;8.6;8.7;8.9;9.0+PTX"
```

Comment:
Please add back the comment explaining why we are doing this :)

Reply:
done